src/oc/audit.cpp

```diff
@@ -40,7 +40,7 @@ maintain a list of xopen statements with the proper rcs version number.
 
 
 static void hoc_audit_init(void) {
-#if !OCSMALL
+#ifndef OCSMALL
```

Comment:
@alexsavulescu : just to be sure - as `if !OCSMALL` != `ifndef OCSMALL`, did you check that there are no usages like `define OCSMALL 0` ?

I quickly checked and I don't see anything. But asking just to be sure...

Reply:
Yes, there was something off before. I had to fix some undefined symbols :)

Reply:
Now MEMACS is no longer tied to OCSMALL

Reply:
We should remove that